docs/bucket/replication/README.md

```diff
@@ -13,14 +13,14 @@ To replicate objects in a bucket to a destination bucket on a target site either
 Create a replication target on the source cluster as shown below:
 
 ```
-mc admin bucket replication set myminio/srcbucket https://accessKey:secretKey@replica-endpoint:9000/destbucket --path ON --api s3v4
+mc admin bucket remote set myminio/srcbucket https://accessKey:secretKey@replica-endpoint:9000/destbucket --path ON --api s3v4 --type "replica"
```

Comment:
we can remove --api s3v4 since s3v4 will be preferred s3v2 is already deprecated doesn't need to be extended to this feature.